index.html

```diff
@@ -2,7 +2,7 @@
 <body>
   <h1>Meet the team</h1>
   <ul>
-    <li>person 1</li>
+    <li><a href="">OLGATRON</a></li>
```

Comment:
This will not actually link anywhere, you should add a file called olga.html and change the href to `/olga.html` here.